pyproject.toml

```diff
@@ -39,7 +39,7 @@ all = [
     'shapely',
 ]
 test = [
-    'pytest-arraydiff',
+    'pytest-arraydiff @ git+https://github.com/astropy/pytest-arraydiff',
```

Comment:
I released v0.6.0 so this is no longer necessary to pull from main
```suggestion
    'pytest-arraydiff',
```

Reply:
done